rollup/go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/holiman/uint256 v1.3.2
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/prometheus/client_golang v1.16.0
-	github.com/scroll-tech/da-codec v0.1.3-0.20250401062930-9f9f53898493
+	github.com/scroll-tech/da-codec v0.1.3-0.20250513170653-750488ce7dfd
```

Comment:
Merge that PR and change commit before merging this one

Reply:
Updated in this commit: [9911d77](https://github.com/scroll-tech/scroll/pull/1658/commits/9911d77d10706d764aece34537c047dd063272ec).